Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM node:8.17.0-jessie as builder
+FROM node:14.8.0-slim as builder
 WORKDIR /usr/build
 
 # Install app dependencies
 COPY package*.json ./
-RUN npm install --production
+RUN yarn install --production
```

Comment:
tbh most of the reasons are outdated as for 2020 between npm vs yarn v1.

- One of the main reasons why facebook created yarn was security. NPM allowed packages to run code on installation. There were many cases with security issues.

previously it was
- Yarn downloads packages in parallel. Nearly 40% faster at installing dependencies. There is further boost on re-installations on local environment because of caching. (p.s the difference may be lesser today due to npm imrovements)
- Its output is cleaner than npm's. This helps in cicd
- Because devs today mostly use yarn, having yarn in docker helps with consistency 
- Has automatic retries in case of network flakiness

will actually revent to use npm in prod. Still yarn locally as the syntax is slightly more convenient